dashboard/app/controllers/scripts_controller.rb

```diff
@@ -261,14 +261,14 @@ def get_unit
 
     script =
       params[:action] == "edit" ?
-      Script.get_without_cache(unit_id, with_associated_models: true) :
```

Comment:
should `script` be renamed to `unit` on line 262 (this might be better as a follow up)

Reply:
To keep the size of this PR in check, I was proposing not renaming local variables yet, but I can certainly do this as part of a follow-up PR.

Reply:
https://codedotorg.atlassian.net/browse/PLAT-2031